src/main/java/io/harness/cf/client/api/analytics/AnalyticsCacheFactory.java

```diff
@@ -9,11 +9,11 @@
 
   public static Cache create(String cacheName) {
     if (cacheName.equals(CAFFEINE_CACHE)) {
-      log.info("Using Caffeine cache");
+      log.debug("Using Caffeine cache");
       return new CaffeineCache();
     }
     if (cacheName.equals(GUAVA_CACHE)) {
-      log.info("Using Guava cache");
+      log.debug("Using Guava cache");
       return new GuavaCache();
```

Comment:
I think it is better to have default cache and if customer want to change cache implementation just need to specify dependency in client builder.